### PR TITLE
support showing multi objects localization marks in RVIZ

### PR DESCRIPTION
**Pull request by ahuizxc**

### CODE REVIEW

File: object_analytics_rviz/src/marker_publisher.cpp [hunk 1]
```diff
@@ -131,11 +131,11 @@ class MarkerPublisher : public rclcpp::Node
 
       std::vector<TrackingObjectInBox> objects_tracked;
       std::vector<LocalizationObjectInBox> objects_localized;
-
       std_msgs::msg::Header header = loc->header;
       objects_tracked = tra->tracked_objects;
       objects_localized = loc->objects_in_boxes;
-
+      if (objects_tracked.size() != objects_localized.size())
```
Comment (yechun1) [inline object_analytics_rviz/src/marker_publisher.cpp]: May I know why objects_tracked size must be the same with objects_localized size?

File: object_analytics_rviz/src/marker_publisher.cpp [hunk 2]
```diff
@@ -145,62 +145,59 @@ class MarkerPublisher : public rclcpp::Node
...
-                      MarkerPublisher::publishMarker(header, box_min, box_max, obj_name, obj_id);
-                  }
-              }
-
-          }
-      }
+    visualization_msgs::msg::MarkerArray marker_array;
+    marker_array.markers = std::vector<visualization_msgs::msg::Marker>();
+    visualization_msgs::msg::Marker marker_clear;
+    marker_clear.action = 3;
```
Comment (yechun1) [inline object_analytics_rviz/src/marker_publisher.cpp]: what does "3" mean? Could be replaced by visualization_msgs::msg::Marker:: ?

File: object_analytics_rviz/src/marker_publisher.cpp (hunk 1 above)
Comment (ahuizxc) [reply object_analytics_rviz/src/marker_publisher.cpp]: can make the 3d markers displaying in RVIZ more smooth in theory.
In most cases, these two variables should be the same.

**Comment by ahuizxc** [comment]
辛苦了！

**Pull request merged**